### ISSUE TITLE
Graphical Text Translations

### ISSUE THREAD

**Issue by Dimedime-d** [Dimedime-d/kptranslation#5]
The following is a list of all instances of graphical Japanese text that I could find:

- [x] Title Screen (2 words)
- [x] Single Player Menu Buttons (~10 words)
- [x] Multiplayer Menu Buttons (~10 words, compressed) (also added to multiboot GBAs)
- [x] In Multiplayer minigames, the ending podiums 1st/2nd/3rd/4th (4 words, compressed) (also changed in multiboot GBAs)
- [x] Overworld levels, minigames and world names (~50 phrases)
- [x] Splash screens for each world (5 phrases)
- [x] Magic option preview (16 paragraphs)
- [x] Magic "Learn" options (2 words)
- [x] Text inside some magic trick helper images (~8 phrases)
- [x] "Random" option in Challenge menus (1 word) (also added to multiboot GBAs)
- [x] Adventure mode minigame splash screens (12 paragraphs)
- [x] Challenge mode minigame splash screens (16 paragraphs)
- [x] Multiplayer minigame splash screens (16 paragraphs)
- [x] "Congratulations" screen when beating Neo Land-4 (1 phrase)
- [x] "Demo" and "Replay" labels in certain game modes
- [x] No record because of shorter propeller text box
- [x] "You got a key!" textbox

~Currently in the process of transcribing this text, not yet on my pastebin...~ Don't really need to transcribe/translate word-for-word, just need to communicate the intent.

**Comment by NewGBAXL** [comment]
is there any way I can contribute?
I've worked on the GameCube Kururin and did graphics, reversing, and translation

**Comment by Dimedime-d** [comment]
Of all the tasks, I would most want help with the final congratulations screen after beating the very last level:
![end3](https://github.com/Dimedime-d/kptranslation/assets/73413313/6297f09d-9b8d-4d4f-9af9-9e9a4a86fc8e)

I've automated much of the process of inserting graphics into the ROM. Starting with an arbitrary .png, I can re-quantize it to the correct palette, then format it into compatible binaries (tileset + tilemap) using grit. Then I find the location of the original tileset/map in the ROM and re-point it to the new ones. The biggest obstacle would be the graphical design needed to produce the image to format in the first place. 

~~All the multiplayer stuff is kind of secondary, considering how few people play that mode with emulation. The weird compression #6 is still an issue.~~ Multiplayer is now done.

~~I'm in the process of recovering some local work with all the magic tricks. The instructions have a lot of (SHIFT-JIS) text to translate and the [previews](https://github.com/Dimedime-d/kptranslation/wiki/Graphical-Text) are pretty annoying to OCR thanks to the tiny characters.~~ Magic is now done.

A lot of the translated graphics (title screen, level names) are placeholders, and I would also appreciate any refinements/corrections to them.

**Comment by Dimedime-d** [comment]
Unless I missed anything, everything should be replaced. Some data are in more constrained spaces, like the payloads sent to MultiBoot GBAs in Single Pak Versus, but everything else for the most part can be referenced in free space.

The above picture has been done: 
![congrats](https://github.com/Dimedime-d/kptranslation/assets/73413313/a47a1c12-0585-4dd5-bd45-e67843365ceb)

Closing for now, as I've figured out how to replace all relevant data. Any refinements are appreciated, as always.
